flowermd/library/systems.py

```diff
@@ -1,3 +1,36 @@
 """Examples for the Systems class."""
 
-# This is a placeholder for any class that inherits from base.system
+import numpy as np
+from scipy.spatial.distance import pdist
+
+from flowermd.base.system import System
+
+
+class SingleChainSystem(System):
+    """Builds a box around a single chain.
+
+    Calculates the maximum distance of the chain using scipy.spatial.distance.pdist().
```

Comment:
We can expand here a little.  We should say that the box lengths are chosen so that they are at least as long as the largest particle part distance. We could say that the chain is centered in the volume.